boa/contracts/vyper/decoder_utils.py

```diff
@@ -14,33 +14,6 @@
 )
 from vyper.utils import unsigned_to_signed
 
-from boa.vm.utils import ceil32, floor32
-
-
-# wrap storage in something which looks like memory
-class ByteAddressableStorage:
```

Comment:
i think this should stay with the other decoder utils, but maybe we can pass it an `evm` instance instead of `accountdb`